test/agent/cmd/traffic-server/main.go

```diff
@@ -40,8 +41,10 @@ func main() {
 		StartTCPServer(addr)
 	} else if serverMode == "udp" {
 		StartUDPServer(addr)
+	} else if serverMode == "http" {
```

Comment:
Nit: Need to update "// Supported modes are tcp, udp" for var serverMode string on line 29